CONVENTIONS.md

```diff
@@ -64,6 +82,7 @@ name patterns you should use.
 ### Ticks and updates
 
 Use "tick" for updates done once per tick. Use "update" for other kind of updates.
+Note that "tick" is used in block-related code for things that are not performed once per tick for historical reasons. For example, neither `randomTick` nor `scheduledTick` gets called once per tick.
```

Comment:
Nit: not solely historical reasons but also game strings (`randomTickSpeed` etc)